pandas/tseries/period.py

```diff
@@ -355,6 +355,37 @@ def freqstr(self):
         return self.freq
 
     def asfreq(self, freq=None, how='E'):
+        """
+        Convert the PeriodIndex to the specified frequency `freq`.
+
+        Parameters
+        ----------
+
+        freq : str
+            a frequency
+        how : str {'E', 'S'}
+            'E', 'END', or 'FINISH' for end,
+            'S', 'START', or 'BEGIN' for start.
+
+        Returns
+        -------
+
+        new: PeriodIndex with the new frequency
+
+        Examples
+        --------
+
+        >>> pidx = pd.period_range('2010-01-01', '2015-01-01', freq='A')
```

Comment:
maybe also show how `pidx` looks like? then it is easier to compare it with the output of ´asfreq´